File: medico/Domain/Medico.Domain/Models/MedicalRecord.cs
using System;

namespace Medico.Domain.Models
{
    public class MedicalRecord : Entity
    {
        public string Notes { get; set; }

        public Patient Patient { get; set; }

        public Guid PatientId { get; set; }

        public DateTime CreateDate { get; set; }

        public string DocumentType { get; set; }

        public bool IncludeNotesInReport { get; set; }
    }
}

File: medico/Services/Medico.Api/ClientApp/src/app/patientChart/models/medicalRecord.ts
export class MedicalRecord {
    id: string;
    notes: string;
    patientId: string;
    documentType: string;
    createDate: any;
    includeNotesInReport: boolean;

    constructor() {
        this.createDate = new Date();
        this.includeNotesInReport = true;
    }
}